containers/monitoring-alertmanager-image/Dockerfile

```
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: MIT
+
+#     Copyright (c) 2025 SUSE LLC
+
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon.
+
+#!BuildTag: uyuni/alertmanager:%%alertmanager_patch_version%%
+#!BuildTag: uyuni/alertmanager:latest
```

this I think  should also include `monitoring-`

```suggestion
#!BuildTag: uyuni/monitoring-alertmanager:latest
```